### ISSUE TITLE
Params for audio encoder

### ISSUE THREAD

**Issue by Elypha** [rigaya/NVEnc#212]
Hello,
How can I check out all params that a specific audio encoder accepts, like the one mentioned in options.md that `--audio-codec aac:aac_coder=twoloop`?

**Edit**
Is it using the same library as (or using actually) FFmpeg? I tried a little bit myself and found most of the options documented [here](https://ffmpeg.org/ffmpeg-codecs.html) worked.
By the way, `--audio-codec libopus ` seems to be not working well... It shows no error at `--log-level=info` but outputs an empty file (while `--audio-codec aac` worked).

**Comment by rigaya** [comment]
> Is it using the same library as (or using actually) FFmpeg? I tried a little bit myself and found most of the options documented here worked.

Exactly, as NVEncC utilizes libavcodec for audio encoding, most of the options for ffmpeg audio encoders should work.

> By the way, --audio-codec libopus seems to be not working well... It shows no error at --log-level=info but outputs an empty file (while --audio-codec aac worked).

Seems to be crushing inside libopus, I might need to check the build.

**Comment by Elypha** [comment]
Thanks! Also, my specs are as the followings in case you may need more info for the `libopus` thing:
- NVEncC_4.68_x64 release
- Windows 10 x64 1909 18363.720 (AMD platform)
```powershell
nvencc64.exe --device 0 --input $raw --codec hevc `
--audio-codec libopus `
--preset quality --profile main --tier high --output-depth 8 --lookahead 32 -b 3 `
--vbr-quality 30 --vbrhq 0 --max-bitrate 2700 `
--output $out
```
The input is a .mp4 file with 1 video stream in AVC and 1 audio stream in AAC, and the output container is .mkv.

**Comment by rigaya** [comment]
I have fixed libopus build, problem with libopus should now be fixed in [NVEnc 4.69](https://github.com/rigaya/NVEnc/releases/tag/4.69).

**Comment by Elypha** [comment]
I tested encoding with 4.69 and the issue was gone. Thanks!
